remoulade/state/backends/postgres.py

```diff
@@ -64,6 +77,25 @@ class StateVersion(Base):
     version = Column(SmallInteger, primary_key=True)
 
 
+def filter_query(query, selected_actors, selected_statuses, selected_ids, start_datetime, end_datetime, selected_type):
```

Comment:
```suggestion
def filter_query(query, *, selected_actors, selected_statuses, selected_ids, start_datetime, end_datetime, selected_type):
```
To force explicitly putting parameter names, as it is easy to change this API and create errors